src/Compilers/Extension/Roslyn.Compilers.Extension.csproj

```diff
@@ -33,6 +33,7 @@
     <PackageReference Include="System.Collections.Immutable" IncludeInVsix="true" />
     <PackageReference Include="System.Memory" IncludeInVsix="true" />
     <PackageReference Include="System.Reflection.Metadata" IncludeInVsix="true" />
+    <PackageReference Include="System.Numerics.Vectors" IncludeInVsix="true" />
```

Comment:
Just wondering whether BuildBoss shouldn't catch cases like this. cc @jaredpar

Reply:
That's my concern as well. Generally BuildBoss should be used to catch inconsistencies like this. That is why I'm curious about what scenarios are not working today.